.travis.yml

```diff
@@ -17,6 +17,10 @@ matrix:
 
   fast_finish: true
 
+  allow_failures:
+      -   php: 7.2
+      -   php: 7.4snapshot
```

Comment:
This is not allowed.
Please fix this as per https://github.com/spryker/rabbit-mq/pull/44 requested last year with highest prio.